src/containers/shared/components/Currency.tsx

```diff
@@ -1,47 +1,33 @@
 import React from 'react'
-import PropTypes from 'prop-types'
-import Account from './Account'
+import { Link } from 'react-router-dom'
 
 // https://xrpl.org/currency-formats.html#nonstandard-currency-codes
 const NON_STANDARD_CODE_LENGTH = 40
 
-interface Props {
+export interface Props {
```

Comment:
Why is this exported?

Reply:
So that PropTypes arent needed.  Otherwise it complains about no default proptypes set.